### PR TITLE
Exclude element tokens on spirit board from cleanup during time passes.

### PR DESCRIPTION
**Pull request by tomprince**

### CODE REVIEW

File: global-script.lua [hunk 1]
```diff
@@ -2152,12 +2152,17 @@ function resetPiece(object, rotation, offset)
 end
 function handlePlayer(color, data)
     local zone = getObjectFromGUID(elementScanZones[color])
+    local spiritBoardTopEdge = getObjectFromGUID(elementScanZones["Red"]).getPosition().z
```
Comment (iakona) [inline global-script.lua:2155]: why not just use `zone` here?

**Comment by iakona** [comment]
overall not sure about this change since it's a destructive change from how to current mod works

File: global-script.lua (hunk 1 above)
Comment (tomprince) [reply global-script.lua]: This is copy paste from an old version of the code (#3) that did not have that variable. Fixed.

**Comment by tomprince** [comment]
> overall not sure about this change since it's a destructive change from how to current mod works

I was catching up on the last couple of days of #spirit-island-mod in discord, and I saw a number of people surprised by cleaning up the elements on the board. I think this is a nice quality-of-life improvement, as it is one less thing people need to learn to do (that is, locking permanent elements on the board).

**Comment by iakona** [comment]
in general the idea behind the mod is that locked things aren't touched during time passes, while unlocked things can be. not sure i want to break away from that sentiment since for example some people may be putting all element tokens on their player board, and now they don't get cleaned up

**Comment by tomprince** [comment]
I think there is a balance between doing what people expect, and behaving the same always. I think it is better here to do what people expect.

**Pull request closed without merging**